erpnext/accounts/doctype/payment_reconciliation/payment_reconciliation.js

```diff
@@ -21,7 +21,45 @@ frappe.ui.form.on("Payment Reconciliation Payment", {
 				doc: frm.doc,
 				method: 'get_difference_amount',
 				args: {
-					child_row: row
+					child_row: row,
+					source_table : 'payments'
+				},
+				callback: function(r, rt) {
+					if(r.message) {
+						frappe.model.set_value(cdt, cdn,
+							"difference_amount", r.message);
+					}
+				}
+			});
+		}
+	}
+});
+
+frappe.ui.form.on("Payment Reconciliation Invoice",{
+	payment_jv_number : function(frm,cdt,cdn){
+		let child = locals [cdt] [cdn];
+		if(child.payment_jv_number){
+			let entry = child.payment_jv_number.split(' | ');
+			let entry_type = entry[0];
+			let entry_number = entry[1];
+
+			let entry_amount = frm.doc.payments.filter((d)=>{
+				return d.reference_name == entry_number && d.reference_type == entry_type;
+			})[0].amount;
+			frappe.model.set_value(cdt,cdn,"payment_jv_amount",entry_amount);
+
+			frm.doc.payments.forEach((row) => {
+				if (entry_number == row.reference_name){
+					frappe.model.set_value(cdt,cdn,"reference_row",row.reference_row);
+				}
+			});
+
+			frm.call({
+				doc: frm.doc,
+				method: 'get_difference_amount',
+				args: {
+					child_row: child,
+					source_table : 'invoices'
```

Comment:
No space before ":"  and single space after it
```suggestion
					source_table: 'invoices'
```